docker/docker-compose.test.yml

```diff
@@ -13,4 +13,4 @@ services:
     environment:
       POSTGRES_HOST_AUTH_METHOD: trust
       POSTGRES_PASSWORD: art
-    image: metabrainz/musicbrainz-test-database:production
+    image: metabrainz/musicbrainz-test-database:v-2024-01-27
```

Comment:
Is it intended? The second commit seems to be unrelated. At least the commit message doesn’t explain it.

Reply:
IIRC there were some changes in the SQL inside the image that were needed. I thought that pinning the tag used as done here made sense, since the production tag is updated whenever we rebuild MB docker images.

Reply:
> IIRC there were some changes in the SQL inside the image that were needed.
OK, was it supposed to be a temporary tag for testing a change that was in another branch?

> the production tag is updated whenever we rebuild MB docker images
Docker Compose will pull the `production` image only if it isn’t locally available, that is for the first run. By default, it won’t pull the image again even if the hub is updated, but it is always possible to ask Docker to check to pull the image again if needed.

Other MeB projects use the tag `production` mainly, or `beta`.

Reply:
Yes, I remember now that it simply needed the t/sql/initial.sql changes from https://github.com/metabrainz/musicbrainz-server/pull/3154 (which is built into the DB test image). But those changes should be in the production tag now (assuming an old version of the tag isn't cached on Jenkins).

I will try reverting the image tag back to production.